src/Components/Settings/Settings.module.scss

```diff
@@ -0,0 +1,53 @@
+@import '../../style/variables.scss';
+@import '../../style/mixins.scss';
+
+.settingsContainer {
+    display: flex;
+    flex-direction: column;
+    width: 30%;
+    margin: auto;
+    background: $darkBlue;
+    color: #F8F8FF;
+    border-radius: 0.7em;
+    padding: 15px;
+}
+
+.settingsHeader {
+    text-align: center;
+    margin: 0;
+}
+
+.intervalContainer {
+    display: flex;
+    flex-direction: row;
+    align-items: center;
+}
+
+.intervalLabel {
+    padding-right: 5px;
+}
+
+.minutesInput {
+    margin-right: 5px;
+    border-radius: 0.25em;
+    padding: 4px;
+    border: none;
+
+    &:focus {
+        outline: 1px solid $salmonHover;
+    }
+}
+
+.breakIntervalDropdown {
+    width: fit-content;
```

Comment:
just FYI you can use the size mixin to replace width & height here!

Reply:
Awesome!! I don't think I had pulled in those changes from `main` when I was writing this originally.